### DISCUSSION TITLE
AttributeError: module 'tonic' has no attribute 'prototype'

### DISCUSSION THREAD

**Discussion by BhaskarMukhoty** [neuromorphs/tonic#261] [Q&A] [answered]
I am using Tonic version 1.3.3. While trying to obtain the NCAR dataset using the following:
     train_set = tonic.datasets.NCARS(save_to=data_dir, train=True)
I got the error: AttributeError: module 'tonic' has no attribute 'prototype'
Any suggestions why this might be happening ?

**Comment by biphasic** [comment, marked as answer]
That's because ncars is a prototype dataset and only available in tonic.protoype.datasets

You'll need torchdata installed in order to use it
